DESCRIPTION

```diff
@@ -28,4 +28,4 @@ License: MIT
 LazyData: true
 Roxygen: list(wrap = FALSE)
 VignetteBuilder: knitr
-RoxygenNote: 6.0.1
```

Comment:
I think you should update your roxygen version to avoid unnecessary changes

Reply:
wait, how did roxygen 5.0.0 get in here?